frontend/assets/javascripts/src/modules/form/address/postcode.js

```diff
@@ -0,0 +1,54 @@
+define([
+    '$',
+    'src/modules/form/validation/actions'
+], function ($, actions) {
+    'use strict';
+
+    var IRELAND_STRING = 'ireland';
```

Comment:
Would be good to document what the country specific changes do / why they are needed.